_layouts/landing.html

```diff
@@ -3,8 +3,9 @@
 	<head>
 		<style>html{visibility: hidden;opacity:0;}</style>
   		{% include head.html %}
-
+      <script src="../js/lazysizes.min.js" async=""></script>
```

Comment:
Nitpick; since there's no build tool handling this you could manually add the version of lazysizes to the filename, to prevent caching issues in the future should you upgrade it.

Reply:
Good one, thanks!